test/auto/tests/runtest.sh

```diff
@@ -2,4 +2,8 @@
 
 systemctl --user restart phonesim
 sleep 15
-exec "/usr/lib/libqofono-qt5/tests/${1?Test executable expected}"
+bits=
+if [ -d /usr/lib64 ]; then
+    bits=64
+fi
+exec "/usr/lib${bits}/libqofono-qt5/tests/${1?Test executable expected}"
```

Comment:
Alternatively wouldn't mind moving the tests to /opt where almost all the other tests are. And even the tests.xml in this case, it seems.

Reply:
Ok, I did that and removed some Qt4 legacy too.